shuttle/tests/basic/dfs.rs

```diff
@@ -155,7 +101,7 @@ fn two_threads() {
     }
 
     // See `two_threads_work` for an illustration of all 16 interleavings.
-    assert_eq!(iterations.load(Ordering::SeqCst), 16);
+    assert_eq!(iterations.load(Ordering::SeqCst), 29);
```

Comment:
Why did this jump to 29? Is this due to double switch which the follow-up PR fixes?  What's the number with that in place?